tests/core/test_lightning_module.py

```diff
@@ -211,3 +212,144 @@ def optimizer_step(self, current_epoch, batch_nb, optimizer, optimizer_idx, clos
     )
 
     trainer.fit(model)
```

Comment:
```suggestion
        limit_val_batches=0,
    )

    results = trainer.fit(model)
    assert results
```
just limiting the validation.